run.sh

```diff
@@ -95,15 +95,21 @@ scrape_loki_metrics() {
 
 generate_report() {
     source .bingo/variables.env
-
+    
     sed -i "s/{{TARGET_ENV}}/$TARGET_ENV/i" $REPORT_DIR/README.md
     $EMBEDMD -w $REPORT_DIR/README.md
 
-    for f in $REPORT_DIR/*/*.gnuplot; do
-      gnuplot -e "set term png; set output '$f.png'" "$f"
+    for f in $REPORT_DIR/*.gnuplot; do
+        gnuplot -e "set term png; set output '$f.png'" "$f"
     done
 }
 
+clean() {
+    cat "$REPORT_DIR/table.md" >> "$REPORT_DIR/README.md"
+    cat "$REPORT_DIR/result.md" >> "$REPORT_DIR/README.md"
+    rm "$REPORT_DIR/result.md" "$REPORT_DIR/table.md"
+}
```

Comment:
This should be a part of `generate_report`. In addition, you only need one file. You can write the table of contents straight into `README.md` and then copy out the results into the readme file. Then delete the results file.

Reply:
agree